src/App.tsx

```diff
@@ -1,24 +1,75 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useRef, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import classNames from 'classnames';
+
+enum FilterType {
+  all = 'all',
```

Comment:
You can remove FilterType into types folder

Reply:
> You can remove FilterType into types folder

I think it's a syntax error.
But yes - move ur enum to a separate directory